media/js/src/simulations/simulation4/simulationFour.jsx

```diff
@@ -83,17 +83,15 @@ export const SimulationFour = () => {
         {
             headerId: 'whatarenonlinearregression',
             title: 'What are non-linear regressions?',
-            content:
-            <>
+            content: <>
                 {progress[stage] < 1 && (
                     <WhatAreNonLinearRegressions
-                        setshowDatasets={setshowDatasets}
-                        setshowRegLine={setshowRegLine}
-                        showDatasets={showDatasets}
+                        setShowPolyDatasets={setShowPolyDatasets}
+                        setShowRegLine={setShowRegLine}
+                        showPolyDatasets={showPolyDatasets}
```

Comment:
We can pass in the new names withe the old props here. For example:

<WhatAreNonLinearRegressions
                        setshowDatasets={setShowPolyDatasets}
                        showDatasets={showPolyDatasets}
                      
So we can minimize these changes

Reply:
Good point